pages/learn/welcome/concepts.md

```diff
@@ -11,7 +11,7 @@ An internet-connected, single-board computer running {{ $names.os.lower }} and p
 
 ### Release
 
-A snapshot of code collected into one or more Docker images.
+A snapshot of the code collected into one or more Docker images.
```

Comment:
Since the page talks definitions, it does seem like a correct usage in the sentence. But to add to this PR, I would recommend adding more things to the concepts page. This page is in need of an update. Let me know if you want to help on this. Thanks!